_data/students.yml

```diff
@@ -179,7 +178,7 @@
 
 - name: Ensanguined Scythe
   github: namphong124563
-  image: namphong124563.jpeg
+  image: namphong124563.jpg
```

Comment:
do not make other changes

Reply:
didnt make them!
 I am sorry I guess it is perhaps because of some local changes made before.. that image optimization stuff